### DISCUSSION TITLE
How to update count after instantiation

### DISCUSSION THREAD

**Discussion by floer32** [TanStack/virtual#340] [Q&A] [answered]
Question about async data and `virtual`. 

Is there a way to update a virtualizer's count after instantiation? Or should I be [re-]instantiating after I know the count? (I'm currently doing the latter)

**Comment by floer32** [comment, marked as answer]
Ah, found an example from the Discord. Thanks @ardeora !

* https://discord.com/channels/719702312431386674/982071252908900382/998372405049298944
* https://stackblitz.com/edit/vitejs-vite-wfw1ab?file=src%2FApp.tsx

**Reply by ardeora** [reply]
Yeah sorry about that haha. I will make a pull request to add these examples to this repo itself. But glad you found the answer :D

**Comment by nidamunir-extreme** [comment]
How to do this in lit? Sorry I did not understand your example.
